scripts/bigquery_billing_report/bigquery_billing_report.py

```diff
@@ -41,7 +43,7 @@ def print_submission_pricing(ws_namespace, ws_name, query_sub_id, query_workflow
 
     print "Gathering pricing data..."
 
-    client = bigquery.Client(dataset_project_name)
+    client = bigquery.Client(bill_query_to_project_name)
```

Comment:
choose what project to run the query within, which is which project is billed for the query

Reply:
:+1: